src/main/java/interface_adapter/club_logged_in/ClubLoggedInViewModel.java

```diff
@@ -0,0 +1,13 @@
+package interface_adapter.club_logged_in;
+
+import interface_adapter.ViewModel;
+
+/**
+ * The view model for when a club is logged in. This holds all the data for the club when it is logged in.
+ */
+public class ClubLoggedInViewModel extends ViewModel<ClubLoggedInState> {
+    public ClubLoggedInViewModel() {
+        super("logged in");
```

Comment:
The viewName of this view model should be "club logged in" to not be confused with the student logged in view.

Reply:
I see. I will change this. Thanks for spotting this!